tests/integration/src/test/resources/python/app_mock.py

```diff
@@ -90,6 +91,23 @@ def check_health(self):
             logging.debug("Done processing health request.")
             return
 
+        def handle_ipc_info(self):
+            logging.debug("Reporting IPC info")
+            df_shm_info = subprocess.check_output(["df", "-m", "/dev/shm"])
+            ipc_ns_info = subprocess.check_output(["stat", "-Lc", "%i", "/proc/self/ns/ipc"])
+
+            self.send_response(200)
+            self.send_header('Content-type', 'text/html')
```

Comment:
Maybe we should use content-type application/text

Also, I believe the header is capitalized, normally: `Content-Type`

Reply:
Yep, done